hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLogImplV2.java

```diff
@@ -203,9 +208,11 @@ public void commitTransactions(
       for (DeleteBlockTransactionResult transactionResult :
           transactionResults) {
         if (isTransactionFailed(transactionResult)) {
+          metrics.incrementDeleteTxFailure();
           continue;
         }
         try {
+          metrics.incrementDeleteTxSuccess();
```

Comment:
thanks for the work! i think we should put this at line 231, because there may comes a retried delete command response, and for the retried response , maybe we should not increase the corresponding metric, or maybe we can add a new metric specially for retried command!

Reply:
@JacksonYao287 , thanks for the review.  One thing is there is no resent command, but resent transactions.  Transactions holds in every command can be different.  When transaction is resent, it's added into numBlockDeletionTransactionSent. So, the reply of the resent transaction should be accountable.  

The number of resent transaction is (numBlockDeletionTransactionCreated - numBlockDeletionTransactionSent).

Reply:
thanks for the explanation ， the changes looks good to me , +1, waiting for a clean CI!